cmd/osbuild-tests/main.go

```diff
@@ -84,6 +85,23 @@ func testCompose(outputType string) {
 	if status != "FINISHED" {
 		log.Fatalf("Unexpected compose result: %s", status)
 	}
+
+	if download {
+		runComposerCLI(false, "compose", "image", uuid.String())
+		defer deleteImage(uuid)
+	}
+}
+
+func deleteImage(uuid uuid.UUID) {
+	files, err := filepath.Glob(uuid.String() + "*")
```

Comment:
We ought to know the right filename, no? I'm wary of deleting files based on a glob.

Reply:
I guess I can try `image-info` to get the exact file name. Didn't know how to do that originally and globbing was the fastest.

Reply:
That won't work, because you need a file name to pass to `image-info`.

It seems to me like `composer-cli` doesn't have a way to output the image name. However, I think that globbing in a directory we know nothing about might lead to bugs later. I've proposed https://github.com/osbuild/osbuild-composer/pull/289, which creates a new temporary directory to work in. If that's accepted, please you use that functionality here as well.